lexoid/core/parse_type/static_parser.py

```diff
@@ -89,15 +89,21 @@ def process_table(table) -> str:
 
     # Convert to DataFrame and handle empty cells
     df = pd.DataFrame(table_data)
+    df.replace("", pd.NA, inplace=True)
```

Comment:
Should we also bump the version for future dev/release?

Reply:
You mean the Lexoid version, yes?

Reply:
ya.